data/posts/react-router-brand-update.md

```diff
@@ -0,0 +1,73 @@
+
+The React Router logo is already an elegant abstraction of what React Router does. For anyone that doesn't know, it's a visual representation of a route tree. The connected nodes are an actively rendered route. There is meaning in the way it flows from top-to-bottom and connects adjacent nodes to present the quickest and cleanest path. If you've read [Matt Brophy's post about Fog of War](https://remix.run/blog/fog-of-war), you can see the obvious similarities.
+
+I admire it's simplicity, but from an execution perspective, there were obvious opportunities for improvement. 
+
+<img alt="The old React Router logo with guidelines showing what to fix" src="/blog-images/posts/react-router-brand-update/logo-before.png" class="m-auto w-4/5 border rounded-md" />
+
+Nodes were misaligned, inconsistently positioned, and upon closer inspection it was easy to see that the curves in their connective tissue were not smooth. While I spent some time exploring alternative logos, I came to realize that fixing these issues were the top priority.  
+
+<img alt="The new React Router logo showing what has changed" src="/blog-images/posts/react-router-brand-update/logo-after.png" class="m-auto w-4/5 border rounded-md" />
+
+## Updating the wordmark
+
+When it came to this piece of the puzzle, the team had a greater appetite for something new. Ever since posting about [merging Remix and React Router](https://remix.run/blog/merging-remix-and-react-router) in May, we've been toying with the idea of merging the brands together visually. In particular, we wondered whether this was an opportunity for the Remix "R" to be integrated, considering the way we've used Founders Grotesk Bold as the logo font for both identities in the past. 
+
+Ultimately, we decided against visually integrating it. The Remix brand will stay separate, free to evolve as we awaken from our nap in 2025 to release new Remix packages. By making this decision, the React Router wordmark wouldn't depend on another logo (and wouldn't limit this exploration either). It allowed us to explore something more specific to React Router. 
+
+<img alt="Three iterations of the React Router wordmark" src="/blog-images/posts/react-router-brand-update/wordmark-iterations.png" class="m-auto w-4/5 border rounded-md" />
+
+Another problem was simply that the words "React Router" were quite long when included in a brand lockup. Instead of creating various vertically and horizontally stacked configurations of the Logo + Wordmark, we wondered if we could provide a single, evergreen solution.
+
+Inspired by the ancient "compact disc" logo, we embraced a method of nesting the word "React" and emphasizing "Router" as the more significant idea. Visually, this was an elegant way of reconciling the long width issue while establishing a more accurate hierarchy for the identity.
```

Comment:
Worth including a link here for the youngins?

Reply:
Or even an image of the Compact Disk logo underneath the line

> When the word "Router" is styled as an outline, you can more clearly see the retro inspiration behind it.

Might be distracting though 🤷

Reply:
I think it might be too distracting to actually include an image of the Compact Disk logo, but I'm down with adding a link!